routes/item/[id].tsx

```diff
@@ -12,53 +13,66 @@ import {
 import {
   type Comment,
   createComment,
+  getAreVotedBySessionId,
   getCommentsByItem,
   getItemById,
   getManyUsers,
   getUserById,
   getUserBySessionId,
-  getVotedItemsByUser,
   type Item,
   type User,
 } from "@/utils/db.ts";
 import { redirect } from "@/utils/http.ts";
 import UserPostedAt from "@/components/UserPostedAt.tsx";
 import { pluralize } from "@/utils/display.ts";
 
+const PAGE_LENGTH = 10;
```

Comment:
Please add a short explainer regarding why this should not exceed 10. Perhaps, there's a GitHub issue in the main https://github.com/denoland/deno repo that could be referenced also.